bosh-stemcell/spec/stemcells/gpu_spec.rb

```diff
@@ -0,0 +1,238 @@
+sync:\*:(\d{5}):0:99999:7:::
+games:\*:(\d{5}):0:99999:7:::
+man:\*:(\d{5}):0:99999:7:::
+lp:\*:(\d{5}):0:99999:7:::
+mail:\*:(\d{5}):0:99999:7:::
+news:\*:(\d{5}):0:99999:7:::
+uucp:\*:(\d{5}):0:99999:7:::
+proxy:\*:(\d{5}):0:99999:7:::
+www-data:\*:(\d{5}):0:99999:7:::
+backup:\*:(\d{5}):0:99999:7:::
+list:\*:(\d{5}):0:99999:7:::
+irc:\*:(\d{5}):0:99999:7:::
+gnats:\*:(\d{5}):0:99999:7:::
+nobody:\*:(\d{5}):0:99999:7:::
+systemd-network:\*:(\d{5}):0:99999:7:::
+systemd-resolve:\*:(\d{5}):0:99999:7:::
+messagebus:\*:(\d{5}):0:99999:7:::
+systemd-timesync:\*:(\d{5}):0:99999:7:::
+syslog:\*:(\d{5}):0:99999:7:::
+_apt:\*:(\d{5}):0:99999:7:::
+_chrony:\*:(\d{5}):0:99999:7:::
+uuidd:\*:(\d{5}):0:99999:7:::
+tcpdump:\*:(\d{5}):0:99999:7:::
+runit-log:!:(\d{5})::::::
+_runit-log:!:(\d{5})::::::
+sshd:\*:(\d{5}):0:99999:7:::
+vcap:(.+):(\d{5}):1:99999:7:::
+nvidia-persistenced:(.+):(\d{5}):1:99999:7:::\Z
+END_SHADOW
+    
+        its(:content) { should match(shadow_match) }
+    end
+    
+    describe file('/etc/group') do
+          its(:content) { should eql(<<HERE) }
+root:x:0:
+daemon:x:1:
+bin:x:2:
+sys:x:3:
+adm:x:4:vcap
+tty:x:5:
+disk:x:6:
+lp:x:7:
+mail:x:8:
+news:x:9:
+uucp:x:10:
+man:x:12:
+proxy:x:13:
+kmem:x:15:
+dialout:x:20:vcap
+fax:x:21:
+voice:x:22:
+cdrom:x:24:vcap
+floppy:x:25:vcap
+tape:x:26:
+sudo:x:27:vcap
+audio:x:29:vcap
+dip:x:30:vcap
+www-data:x:33:
+backup:x:34:
+operator:x:37:
+list:x:38:
+irc:x:39:
+src:x:40:
+gnats:x:41:
+shadow:x:42:
+utmp:x:43:
+video:x:44:vcap
+sasl:x:45:
+plugdev:x:46:vcap
+staff:x:50:
+games:x:60:
+users:x:100:
+nogroup:x:65534:
+systemd-journal:x:101:
+systemd-network:x:102:
+systemd-resolve:x:103:
+crontab:x:104:
+messagebus:x:105:
+systemd-timesync:x:106:
+input:x:107:
+sgx:x:108:
+kvm:x:109:
+render:x:110:
+syslog:x:111:
+_chrony:x:112:
+netdev:x:113:
+uuidd:x:114:
+tcpdump:x:115:
+_ssh:x:116:
+runit-log:x:999:
+_runit-log:x:998:
+admin:x:997:vcap
+vcap:x:1000:syslog
+bosh_sshers:x:1001:vcap
+bosh_sudoers:x:1002:
+nvidia-persistenced:x:117:
+HERE
+        end
+    
+    describe file('/etc/gshadow') do
+          its(:content) { should eql(<<HERE) }
+root:*::
+daemon:*::
+bin:*::
+sys:*::
+adm:*::vcap
+tty:*::
+disk:*::
+lp:*::
+mail:*::
+news:*::
+uucp:*::
+man:*::
+proxy:*::
+kmem:*::
+dialout:*::vcap
+fax:*::
+voice:*::
+cdrom:*::vcap
+floppy:*::vcap
+tape:*::
+sudo:*::vcap
+audio:*::vcap
+dip:*::vcap
+www-data:*::
+backup:*::
+operator:*::
+list:*::
+irc:*::
+src:*::
+gnats:*::
+shadow:*::
+utmp:*::
+video:*::vcap
+sasl:*::
+plugdev:*::vcap
+staff:*::
+games:*::
+users:*::
+nogroup:*::
+systemd-journal:!::
+systemd-network:!::
+systemd-resolve:!::
+crontab:!::
+messagebus:!::
+systemd-timesync:!::
+input:!::
+sgx:!::
+kvm:!::
+render:!::
+syslog:!::
+_chrony:!::
+netdev:!::
+uuidd:!::
+tcpdump:!::
+_ssh:!::
+runit-log:!::
+_runit-log:!::
+admin:!::vcap
+vcap:!::syslog
+bosh_sshers:!::vcap
+bosh_sudoers:!::
+nvidia-persistenced:!::
+HERE
+        end
+    end
+end
```

Comment:
Because this stemcell variant is based on Jammy, is there any reasonable way to express the contents of these user/group files as additions to the file definitions in the Jammy test, rather than duplication of all of the content of those files?